examples/datachannel-filexfer/filexfer-ws-signaling.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import asyncio
+import logging
+import time
+import sys
+import os
+
+# from os import path
```

Comment:
No. If you are running aiortc from a git checkout just run from the root of the project:

`pip install -e .`

Reply:
ok.